tmt/base.py

```diff
@@ -1437,7 +1436,11 @@ def name_filter(nodes):
                 if any([re.search(name, node.name) for name in cmd_line_names])]
 
         if Test._opt('source'):
-            tests = [Test(test) for test in self.tree.prune(keys=keys, sources=cmd_line_names)]
+            tests = [
+                Test(
+                    node=test) for test in self.tree.prune(
```

Comment:
A bit weird line break?

Reply:
Blame autopep8 and/or its `--aggressive option :)

Reply:
Fixed.